src/lib.rs

```diff
@@ -337,13 +345,26 @@ impl Message {
     pub fn from_slice(data: &[u8]) -> Result<Message, Error> {
         match data.len() {
             constants::MESSAGE_SIZE => {
+                let secp = Secp256k1::without_caps();
```

Comment:
Hmm, I know this is supposed to become free once upstream has a static context, but I'd like to at least see concept ACKs on that happening upstream before we merge something that relies on it.